app/components/Nav.jsx

```diff
@@ -24,8 +24,8 @@ class Nav extends Component {
             { children }
           </div>
           <div>
-            <span className={searchActive ? "link active" : "link"} onClick={ this.props.activateSearch }>Search</span>
-            <a className="link" href="/map">Map</a>
+            <span className={searchActive ? "link active" : "link"} onClick={ this.props.activateSearch }><img className="icon" src={ `/images/nav/icon-search.svg` } />Search</span>
```

Comment:
These "src" properties don't need to have the curly braces or back ticks. Curly braces are only required for non-string values, and backticks should only be used when using variables in the string (ie. `/images/nav/icon-${variable}.svg`)

Can you change these lines to simple strings, like: `<img className="icon" src="/images/nav/icon-search.svg" />`